### ISSUE TITLE
Writing Name in Database

### ISSUE THREAD

**Issue by tfeige91** [expfactory/expfactory.github.io#18]
I just configured Expfactory to write Data into a mysql-Database. 
Everything works good so far. 

The participant-Table shows _NULL_ in the name column.

![Bildschirmfoto 2020-10-08 um 13 46 13](https://user-images.githubusercontent.com/8275704/95454380-aa1bd200-096c-11eb-960a-9d3dddab5921.jpg)

Is it possible to have the name the participant enters in the expfactory main page to be stored here?

The reason is that we want a longitudinal measure and need to trace the corresponding participants. 
Also we don't want to preregister users with user variables an tokens (I am afraid it's too complicated for the participants as they are elderly persons).

**Comment by vsoch** [comment]
See #17 I answered your question there!

**Comment by vsoch** [comment]
TLDR for readers here: storing PHI in a database is not recommended, and so expfactory does not support it by default. The suggested approach is to ask what data collection methods are approved by your institution, and then use that to collect demographic information, and then issue a token for expfactory. If you are allowed to use the server hosting expfactory, then you can add experiment variables or a specific experiment just for collecting demographics. Expfactory won't do this for you by default, taking a (inmho better) more conservative approach.
